### ISSUE TITLE
postImageToSlack

### ISSUE THREAD

**Issue by michaelpryor** [bstancil/mode-chrome-extension#1]
I'm guessing you are still working on this function.  Doesn't look like it is called from anywhere and the function contents look like test code.  Is that right?

**Comment by bstancil** [comment]
Yeah, this was somewhat experimental. Chrome lets you take screen captures of tabs, which the extension does. The image is returned as a base 64 string, which you can pretty easily upload to a third party, convert to different format, and upload to slack or email; trouble is, that requires uploading to a third party, which we don't want to do. And Slack and most email clients don't support base 64 images directly (though oddly, the iOS email client does, so supporting that is pretty easy..).
